Thunder/bot/plugins/admin.py

```
@@ -234,8 +234,17 @@ async def list_authorized_command(client: Client, message: Message):
     
     text = MSG_ADMIN_AUTH_LIST_HEADER
     for i, user in enumerate(users, 1):
+        display_name = "Unknown"
+        try:
+            tg_user = await client.get_users(user['user_id'])
+            display_name = f"@{tg_user.username}" if tg_user.username else tg_user.first_name or "Unknown"
+        except Exception:
+            pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)